nautobot/extras/tests/test_notes.py

```diff
@@ -0,0 +1,54 @@
+from django.contrib.auth import get_user_model
+from django.contrib.contenttypes.models import ContentType
+from django.db.models.query import QuerySet
+
+from nautobot.dcim.models import Site
+from nautobot.extras.models import Notes
+from nautobot.utilities.testing import TestCase
+
+
+User = get_user_model()
+
+
+class NotesModelTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+
+        cls.site_ct = ContentType.objects.get_for_model(Site)
+        cls.notes_ct = ContentType.objects.get_for_model(Notes)
```

Comment:
Unused?

Reply:
Possibly, I changed these tests many times when trying to decide how to structure them.

Reply:
```suggestion
```